code/webauthn/frontend/utils/string.ts

```diff
@@ -0,0 +1,63 @@
+export function hexToBase64Url(hex: string): string {
```

Comment:
I also had to build similar helpers when working with custom signatures. Since we're already maintaining the SmartAccount extension class for ethers/viem, it might be better to just include some of these there if they are always required when manipulating binary signature data.